### ISSUE TITLE
Feature request|  To find all resource creation blocked by kyverno if possible 

### ISSUE THREAD

**Issue by iamraj007** [fjogeleit/policy-reporter-cli#2]
Not sure if this is possible, if i can get all list of all resource  kyverno policy have blocked while creating..

Like no allowPrivilegeEscalation is set to false (enforce  mode), will be block the creation if at all any user try to

```
kubectl get clusterpolicies.kyverno.io
NAME                                BACKGROUND   ACTION    READY
restrict-privilege-escalation       true         enforce   true
```

![image](https://user-images.githubusercontent.com/47947075/188655943-7a841c48-0a52-4427-bae3-dc2dab5ff594.png)

CLI to find this will be cool feature

The idea is to show : 
- The power of this tool
- Visibility to management..
- User is able to see right away if his recent deployment if failing because of   "blocked" event, without looking at logs..

**Comment by fjogeleit** [comment]
This is already possible with the Policy Reporter Kyverno Plugin. It has an option to create PiolicyReports out of the Kubernetes events of blocked resources.

you can checkout the documentation for details

https://kyverno.github.io/policy-reporter/guide/helm-chart-core#enable-enforce-violation-policyreports-requires-kyverno--170

**Comment by iamraj007** [comment]
Also such output(below) don't have timestamp on it, do bit difficult to a big environment  

![image](https://user-images.githubusercontent.com/47947075/188662119-28d174e2-ae93-4fce-8af9-c7dbd2d8ed21.png)

**Comment by fjogeleit** [comment]
I can add the timestamp to the wide output format

**Comment by iamraj007** [comment]
> I can add the timestamp to the wide output format

That will do, thanks :)

**Comment by fjogeleit** [comment]
Version 0.5.0 has the created time in the wide output but the timestamp was for now not part of the API. I added it to the rest API and tagged a version 2.9.0 for policy reporter. I will release it tomorrow I think but you can overwrite the tag in the helm chart after the build is done.
